### DISCUSSION TITLE
Question: What gUFO properties express mereology?

### DISCUSSION THREAD

**Discussion by plbt5** [nemo-ufes/gufo#32] [Q&A] [answered]
I'd like to get a clear picture on mereology in gUFO, and as a starter I'd like to get an overview of the (object)properties that are typically used to express that phenomenon. 

In reference to the [article on mereology](https://plato.stanford.edu/entries/mereology/) from the Stanford Encyclopedia of Philosophy, the core principles for parthood are characterized by relations that are either Reflexive, Transitive or Antisymmetric or combinations thereof, and of course relate parts with their wholes. I'm not sure but suggest the following ObjectProperties to be part of (pun not intended) such list of mereological properties:

- [concernsConstitutedEndurant](https://nemo-ufes.github.io/gufo/#concernsConstitutedEndurant)
- [concernsTemporaryWhole](https://nemo-ufes.github.io/gufo/#concernsTemporaryWhole)
- [constitutes](https://nemo-ufes.github.io/gufo/#constitutes)
- [isAspectProperPartOf](https://nemo-ufes.github.io/gufo/#isAspectProperPartOf)
- [isComponentOf](https://nemo-ufes.github.io/gufo/#isComponentOf)
- [isEventProperPartOf](https://nemo-ufes.github.io/gufo/#isEventProperPartOf)
- [isObjectProperPartOf](https://nemo-ufes.github.io/gufo/#isObjectProperPartOf)
- [isProperPartOf](https://nemo-ufes.github.io/gufo/#isProperPartOf)
- [isSituationProperPartOf](https://nemo-ufes.github.io/gufo/#isSituationProperPartOf)
- [gufo:isSubQuantityOf](https://nemo-ufes.github.io/gufo/#isSubQuantityOf)
- [standsInQualifiedConstitution](https://nemo-ufes.github.io/gufo/#standsInQualifiedConstitution)
- [standsInQualifiedParthood](https://nemo-ufes.github.io/gufo/#standsInQualifiedParthood)

**Q1:** Please correct me where I'm wrong, and add properties that I've missed.

Thanks in advance for your support.

**Comment by ManuelSimoes** [comment]
I'd be interested to know if _constitutes_ is considered a mereological property. The article on mereology from the Stanford Encyclopedia of Philosophy lists this one as "controversial," and in the gUFO documentation only gufo:isProperPartOf and its subproperties are mentioned as being part-whole relations.
One of the examples given for constitutes, "the situation in which the Beatles is constituted by the group composed of {John,Paul,Ringo,George}", suggests a difference between the Beatles being _constituted (non-mereologically)_ by the group, and the group being _composed (mereologically)_ by John, Paul, Ringo and George.

**Reply by jpalmeida** [reply]
We assume Constitution is not a mereological property, as it does not satisfy the weak supplementation principle (following [1]). You are right in your interpretation of the Beatles example!

[1] Simons, P.M. Parts. An Essay in Ontology, Oxford: Clarendon Press, 1987.

**Comment by jpalmeida** [comment, marked as answer]
> In reference to the [article on mereology](https://plato.stanford.edu/entries/mereology/) from the Stanford Encyclopedia of Philosophy, the core principles for parthood are characterized by relations that are either Reflexive, Transitive or Antisymmetric or combinations thereof, and of course relate parts with their wholes. 

Reflexivity depends on whether we are talking about proper or improper parthood. In gUFO, we have gufo:isProperPart, so, it is not reflexive (and neither are its sub-properties)!

Concerning transitivity, gufo:isProperPart itself is transitive. But not all sub-properties are. For example, gufo:isComponentOf is transitive only in some cases, as discussed in [1], and gufo:isCollectionMemberOf is never transitive [1].

gufo:isProperPart is not marked Asymmetric because that would violate structural constraints in OWL-DL. It would become a "non-simple property". So, it is one of these cases in which we can't express all we wanted to express in OWL-DL.

[1] https://research.utwente.nl/en/publications/ontological-foundations-for-structural-conceptual-models
